icon/Icon.js

```diff
@@ -166,18 +167,21 @@ export const Icon = {
     folder(p)           {return Icon.icon({...p,  iconName: 'folder'})},
     folderOpen(p)       {return Icon.icon({...p,  iconName: 'folder-open'})},
     fund(p)             {return Icon.icon({...p,  iconName: 'university'})},
+    function(p)         {return Icon.icon({...p,  iconName: 'function'})},
```

Comment:
this looks a bit hairy, w.r.t to function being a key word -- but I think
 its ok!

Reply:
could call it func() if we wanted to, to avoid any confusion with our tooling.